e2e-tests/test/main.test.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { Wallet, Contract } from 'zksync-web3';
+import * as hre from 'hardhat';
+import { Deployer } from '@matterlabs/hardhat-zksync-deploy';
+
+const RICH_WALLET_PK =
```

Comment:
```suggestion
// In-memory node rich wallet
const RICH_WALLET_PK =
```

Reply:
Defer this change as it's optional and technically the same PK we have for local setup also